module/network/PlotJuggler/data/config/PlotJuggler.yaml

```diff
@@ -0,0 +1,12 @@
+log_level: INFO
+
+# Set to true to forward DataPoint messages to PlotJuggler
+forward_datapoints: true
+
+# Set to true to generate and emit sample datapoints (for testing the connection to PlotJuggler)
+send_debug_waves: true
```

Comment:
Seems like it might be annoying to have this on by default? Or is it not really intrusive in PlotJuggler?

Reply:
Good catch, it should be false by default.